.kitchen.yml

```diff
@@ -8,16 +8,21 @@ driver:
 platforms:
 - name: centos-6
   driver:
-    box: chef/centos-6.6
+    box: chef/centos-6.6  # vagrant
+    image: centos-6-5-x64 # digitalocean
```

Comment:
Can DO-specific configuration be dedicated to `.kitchen.digitalocean.yml` ? Or we have to keep this image name only in `.kitchen.yml`?

P.s. Sorry, I haven't worked with another kitchen drivers except vagrant, that's why I'm asking :)